addons/account/tests/test_account_payment_register.py

```diff
@@ -2,8 +2,9 @@
 from odoo.addons.account.tests.common import AccountTestInvoicingCommon
 from odoo.exceptions import UserError
 from odoo.tests import tagged
-from odoo import fields, Command
 
+from odoo.tests.common import Like
+from odoo import fields, Command
```

Comment:
Wrong convention of import here https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports

Reply:
the imports convention was already broken before I step in
I will fix it all (not just my imports) if it is fine for u